strings/redpill.json

```diff
@@ -0,0 +1,18 @@
+{
+	"redpill_questions": [
+		"What happened to the ceiling?",
+		"Why is it called the emergency shuttle if it arrives every single shift?",
+		"Where does the Cook get all this meat from?",
+		"Space wind? How does that even make sense?",
+		"Why does Nanotrasen hire Clowns and Mimes for every single station?",
+		"Why is the station's air supply connected to the plasma tank?",
+		"Why are there fire alarms everywhere but no sprinklers?",
+		"Why is this a plasma research station if we know everything about plasma already?",
+		"Who hires all these unqualified, violent assistants?",
```

Comment:
Why is there civilians on a research station?

Reply:
Yeaah I removed the line because toolboxy assistants is more a tg thing